cheroot/connections.py

```diff
@@ -136,9 +136,11 @@ def get_conn(self, server_socket):
         try:
             for fno in socket_dict:
                 self._selector.register(fno, selectors.EVENT_READ)
+            # The timeout value impacts performance and should be carefully
+            # chosen (issue #305)
```

Comment:
I prefer having a complete URL in the comment (this is friendlier to envs outside of GH)
```suggestion
            # chosen. Ref:
            # github.com/cherrypy/cheroot/issues/305#issuecomment-663985165
```